tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
To verify that parser is working, you can probably just use Esprima and pass it some ES6 that it doesn't yet support. Or JSX, and make sure it fails. I don't think exposing parser within context is a good idea.